addons/website_form/static/src/snippets/s_website_form/options.js

```diff
@@ -751,6 +753,19 @@ options.registry.WebsiteFieldEditor = FieldEditor.extend({
     // Options
     //----------------------------------------------------------------------
 
+    /**
+     * Add a description to the field input
+     */
+    appendDescription: async function (previewMode, value, params) {
+        const description = this.$target[0].querySelector('.s_website_form_field_description');
+        if (description && value) {
+            description.textContent = value;
+        } else {
+            const field = this._getActiveField();
+            field.description = value;
+            await this._replaceField(field);
+        }
```

Comment:
Meh. I think we should be consistent. If I understand the code right here, you probably have a textarea in the left panel to handle this? For the labels I am ok with it but for the description, I feel it is important to be able to put a word in bold or something like that. Can't we do exactly as for the product catalog snippet ? A checkbox in the left panel, and edition in the editor? Technical problem with that?

Reply:
Not a problem to do that I guess. Would not be needed if bold, underline and italic were working on contenteditable="false" elements like all the other header functionalities do.

Reply:
I don't understand your question? Why would have a non editable content editable? And what is the relation with this?